workflow/local_workflow.py

```diff
@@ -268,45 +259,26 @@ def _chunk_iterator(X_array, folder, y_array=None, chunk_size=1024, n_jobs=8,
     vary according to examples (hence the fact that X is a list instead of
     numpy array).
     """
-    for i in range(0, len(X_array), chunk_size):
-        X_chunk = X_array[i:i + chunk_size]
-        filenames = [
-            os.path.join(folder, '{}.{}'.format(x, img_file_extension))
-            for x in X_chunk]
-        X = Parallel(n_jobs=n_jobs, backend='threading')(delayed(imread)(
-            filename) for filename in filenames)
-        if y_array is not None:
-            y = y_array[i:i + chunk_size]
+    for i in range(0, src.shape[0], chunk_size):
+        # X_chunk = X_array[i:i + chunk_size]
+        # filenames = [
+        #    os.path.join(folder, '{}.{}'.format(x, img_file_extension))
+        #    for x in X_chunk]
+        # X = Parallel(n_jobs=n_jobs, backend='threading')(delayed(imread)(
+        #    filename) for filename in filenames)
+        X = src[i:i + chunk_size, :, :]
+
+        if labels is not None:
+            # y = y_array[i:i + chunk_size]
+            y = [list(labels[labels.id == '77_{0}'.format(j)][
+                          ['x_p', 'y_p', 'radius_p']].itertuples(name=None,
+                                                                 index=False))
+                 for j in range(i, i + chunk_size)]
```

Comment:
Same comment as previous

Reply:
This can actually be removed now, but will clarify it in the other occurence